pkg/remotedialer/server.go

```diff
@@ -56,7 +56,7 @@ func (s *Server) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	logrus.Infof("Handling backend connection request [%s]", clientKey)
 
 	upgrader := websocket.Upgrader{
-		HandshakeTimeout: 5 * time.Second,
+		HandshakeTimeout: 60 * time.Second,
```

Comment:
The timeout of 5 seconds is fairly intentional. This the timeout to just do the initial connection handshake of 101.  If it doesn't complete in about 5 seconds, then that means the round trip of about two TCP messages is too slow for the connection to be usable anyhow.